code/modules/projectiles/guns/ballistic/pistol.js

```diff
@@ -24,5 +24,83 @@ module.exports.templates = {
 			icon_state: "pistol"
 		},
 		tree_paths: ["items/gun/ballistic/automatic/pistol"]
+	},
+	"deagle": {
+		components: ["BallisticGun", "BeltItem"],
+		vars: {
+			components: {
+				"BallisticGun": {
+					spawn_mag: "mag_m50",
+					mag_caliber: ".50",
+					mag_form_factor: "pistol",
+					tactical_reloads: true,
+					reload_sound: 'sound/weapons/autoguninsert.ogg',
+					can_suppress: false,
+					mag_state: true
+				},
+				"Examine": {
+					desc: "A robust .50 AE handgun."
+				},
+				"Item": {
+					size: 2
+				},
+			},
+			name: "improper Desert Eagle",

```

Comment:
Remove `improper`

Reply:
yeah specifically the name should just be all lowercase.